content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -1,53 +1,64 @@
-============================
-Manage Amazon orders in Odoo
-============================
+=======================
+Amazon order management
+=======================
 
-Synchronization of orders
-=========================
+Order synchronization
+=====================
+
+Orders are automatically fetched from Amazon, and synchronized in Odoo, at regular intervals.
 
-Orders are automatically fetched from Amazon and synchronized in Odoo at regular intervals.
 The synchronization is based on the Amazon status: only orders whose status has changed since the
-last synchronization are fetched from Amazon. For **FBA** (Fulfilled by Amazon), only **Shipped**
-and **Canceled** orders are fetched. For **FBM** (Fulfilled by Merchant), the same is done for
-**Unshipped** and **Canceled** orders. For each synchronized order, a sales order and a customer are
-created in Odoo if they are not yet registered.
+last synchronization are fetched from Amazon.
+
+For *FBA* (Fulfilled by Amazon), only *Shipped* and *Canceled* orders are fetched.
 
-When an order is canceled in Amazon and was already synchronized in Odoo, the corresponding sales
-order is automatically canceled in Odoo.
+For *FBM* (Fulfilled by Merchant), the same is done for *Unshipped* and *Canceled* orders. For each
+synchronization order, a sales order and customer are created in Odoo, if they aren't already
+registered.
 
 .. note::
-   To force the synchronization of an order whose status has not changed since the last
-   synchronization, activate the :ref:`developer mode <developer-mode>`, navigate to your Amazon
-   account and modify the date under :menuselection:`Orders Follow-up --> Last Order Sync`. Pick a
-   date anterior to the last status change of the order that you wish to synchronize and save.
+   When an order is canceled in Amazon, and was already synchronized in Odoo, the corresponding
+   sales order is automatically canceled in Odoo.
+
+Force synchronization
+=====================
+
+In order to force the synchronization of an order, whose status has **not** changed since the
+previous synchronization, start by activating the :ref:`developer mode <developer-mode>`.
+
+Then, navigate to the Amazon account, and modify the date under :menuselection:`Orders Follow-up
```

Comment:
Great point. Given the context, I'd assume this would occur in Odoo, because being in developer mode was the first step. So, I'll add a couple of clarifying mentions that it takes place in Odoo, so as to avoid any confusion. Thanks for bringing this to my attention. Again, my apologies for the lack of total clarity with these amazon docs -- I had to do my best to create them without any access to a paid-for Amazon Seller account.